src/main/java/spotify/factories/AbstractPlayableObjectDeserializer.java

```diff
@@ -7,7 +7,23 @@
 
 import java.lang.reflect.Type;
 
+/**
+ * This class is a custom {@link JsonDeserializer} which deserializes json and returns a {@link AbstractPlayableObject} subclass based on the type property in the json
+ * @see spotify.models.generic.AbstractPlayableObject
+ * @author Jiankai Zheng (jk.zheng@hotmail.com)
+ * @since // check with Jiankai
+ */
+
 class AbstractPlayableObjectDeserializer implements JsonDeserializer<AbstractPlayableObject> {
+
+    /**
+     * This function takes a {@link JsonElement}  and returns an {@link AbstractPlayableObject}  based off the type specified
```

Comment:
There is a double space between `{@link JsonElement}` and `and`, also between `{@link AbstractPlayableObject}` and `based`.

It should also be 'based on the specified type'.

Reply:
OK. Thanks. I will update it.